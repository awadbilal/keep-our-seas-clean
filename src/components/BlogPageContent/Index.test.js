import React from "react";
import renderer from "react-test-renderer";
import BlogPageContent from "./Index";

it("renders correctly", () => {
  const tree = renderer.create(<BlogPageContent />).toJSON();
  expect(tree).toMatchSnapshot();
});
