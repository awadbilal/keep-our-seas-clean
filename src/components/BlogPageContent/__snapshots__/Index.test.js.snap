// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    className="row"
  />,
  <div
    className="justify-content-center row"
  />,
]
`;
